consensus/spos/sposFactory/sposFactory.go

```diff
@@ -12,48 +12,10 @@ import (
 	"github.com/multiversx/mx-chain-go/consensus/broadcast"
 	"github.com/multiversx/mx-chain-go/consensus/spos"
 	"github.com/multiversx/mx-chain-go/consensus/spos/bls"
-	"github.com/multiversx/mx-chain-go/outport"
 	"github.com/multiversx/mx-chain-go/process"
 	"github.com/multiversx/mx-chain-go/sharding"
 )
 
-// GetSubroundsFactory returns a subrounds factory depending on the given parameter
-func GetSubroundsFactory(
-	consensusDataContainer spos.ConsensusCoreHandler,
-	consensusState *spos.ConsensusState,
-	worker spos.WorkerHandler,
-	consensusType string,
-	appStatusHandler core.AppStatusHandler,
-	outportHandler outport.OutportHandler,
-	sentSignatureTracker spos.SentSignaturesTracker,
-	chainID []byte,
-	currentPid core.PeerID,
-	signatureThrottler core.Throttler,
-) (spos.SubroundsFactory, error) {
-	switch consensusType {
-	case blsConsensusType:
```

Comment:
we used `blsConsensusType`, now we have other versioning, do we need `blsConsensusType` const anymore? or maybe considering blsConsensusV1 and 2 for newly added versions

Reply:
The type here was for the type of votes used during consensus.
both versions use bls signatures as votes.
If we change to something else we could add the switch again, until then I don't think we need it